drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -92,13 +96,13 @@ def with_household_tasks(scope)
       SQL
 
       # Clause for checking whether an Assessment falls within the "due period". There are due cases because the due period may be this year or last year.
```

Comment:
```suggestion
      # Clause for checking whether an Assessment falls within the "due period". There are two cases because the due period may be this year or last year.
```